model/election.go

```diff
@@ -0,0 +1,117 @@
+package model
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/vocdoni/blind-csp/types"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo/options"
+	"go.vocdoni.io/dvote/log"
+)
+
+var ErrElectionUnknown = fmt.Errorf("election is unknown")
+
+type HandlerConfig struct {
+	Handler string   `json:"handler" bson:"handler"`
+	Service string   `json:"service" bson:"service"`
+	Mode    string   `json:"mode" bson:"mode"`
+	Data    []string `json:"data" bson:"data"`
+}
+
+type Election struct {
+	ID       types.HexBytes  `json:"electionId" bson:"_id"`
+	Handlers []HandlerConfig `json:"handlers" bson:"handlers"` // List of handlers that will use this census
+}
+
+type ElectionStore interface {
+	Create(election *Election) (*Election, error)
+	Election(id types.HexBytes) (*Election, error)
+	Delete(id types.HexBytes) error
+	List() (*[]types.HexBytes, error)
```

Comment:
same issue as the users interface - IMO these methods should all include the name "election" in the names